server/routes.js

```diff
@@ -7,12 +7,31 @@ router.get('/friendBoard', controllers.leaderBoard.getFriendBoard)
 router.get('/globalBoard', controllers.leaderBoard.getGlobalBoard)
 // get market data
 router.get('/order', controllers.orderData.postOrderData)
-router.post('/test1', (req, res) => {
-  // console.log(req.body, 'ahha');
 
-  //res.send('');
+router.get('/status', (req, res) => {
+  console.log('in status');
+  res.send(req.session.userid);
```

Comment:
Consider defensive programing and allowing undefined and null values to fail gracefully using optional chaining https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining